modules/appeals_api/spec/models/higher_level_review_spec.rb

```diff
@@ -221,82 +221,64 @@
                          'Validation failed: Status is not included in the list')
     end
 
-    it 'emits an event' do
-      handler = instance_double(AppealsApi::Events::Handler)
-      allow(AppealsApi::Events::Handler).to receive(:new).and_return(handler)
-      allow(handler).to receive(:handle!)
-
-      higher_level_review.update_status!(status: 'pending')
-
-      expect(handler).to have_received(:handle!)
+    context 'when incoming and current statuses are different' do
+      it 'enqueues the status updated job' do
+        assert_equal 0, AppealsApi::AppealReceivedJob.jobs.size
```

Comment:
`assert_equal` is a different testing framework (Rails' TestUnit) than rspec. Please update to use `expect` syntax.

Reply:
Also, shouldn't this be making sure that `StatusUpdatedJob`'s size changed, not `AppealReceivedJob`? I'd personally add in "extra" expectations around both types of jobs to ensure they're both in the state you expect, since none/either/both can be created given different situations.

Reply:
The calling of the same job over and over is another drafty derp. As for `assert_equal`, I'll go back to expect, but there were some strange side effects I was trying to avoid and this resolved it. I got excited after researching sidekiq testing. not gonna like.